src/js/themes/base.js

```diff
@@ -962,6 +962,15 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     },
     select: {
       // background: undefined,
+      clear: {
+        container: {
+          pad: 'small',
+          background: 'light-2',
+        }, // any box props
+        text: {
+          color: 'dark-3',
+        }, // any text props
```

Comment:
Let's cross these colors with Jet's designs and see whether we can make the theme not default unless a must.

Reply:
I used these designs for the default colors. 

<img width="1228" alt="65728693-eee20d00-e078-11e9-829e-cce013916dfb" src="https://user-images.githubusercontent.com/21990914/91894063-b3719a80-ec5a-11ea-8684-3c7925cde6ed.png">

Reply:
I've been discussing the usage of the colors with eric, and we should avoid the usage of `dark-*` and `light-*` colors. We should be using generic coloring like`background-contrast` and `text`/`text-weak` instead.
It might need some exploration of what are the right colors to pick, and it would be OK if we wouldn't have the exact coloring as the designs show, as long as we are maintaining the standard of using generic colors on our default theme.
The light /dark colors are intended to be deprecated at one point, so less work for us later if we remove it from here now.

Reply:
Interesting, it should work. have you tried other colors, maybe `background-front`? I'm suspecting background-contrast might not work on a white background, see if that is the reason.

Reply:
that doesnt work either. just to make sure, this is what i have.

```
    select: {
      // background: undefined,
      clear: {
        container: {
          pad: 'small',
          background: 'background-front',
        }, // any box props
        text: {
          color: 'text-weak',
        }, // any text props
      },
```

Reply:
Yea. that should work, take a look at what colors are 'text-weak' and  'background-*' are being resolved to.

Reply:
text-weak : #555555
background-front: #FFFFFF08

Reply:
@taysea can you help with figuring the color issue?

Reply:
Yes, let me have a look.

Reply:
Hey @IanKBovard, it looks like for the background-contrast adds, you need to invert light and dark values, I used this local on your branch and things worked as expected:
```
'background-contrast': {
    dark: '#FFFFFF08',
    light: '#33333308',
  },
```
<img width="322" alt="Screen Shot 2020-09-03 at 2 54 01 PM" src="https://user-images.githubusercontent.com/12522275/92177427-53fbc200-edf5-11ea-88c7-e5615fd91c4c.png">